main.py

```diff
@@ -95,7 +101,7 @@
 
     elif player["x"] == 1 and player["y"] == 0:
         which_direction = input(
-            "You can go west. There is a map on the ground. (type T to take the map, press M to view it when you have it.)")
+            "You can go east. There is a map on the ground. (type T to take the map, press M to view it when you have it.)")
```

Comment:
```suggestion
            "You can go west. There is a map on the ground. (type T to take the map, press M to view it when you have it.)")
```
For this game, west is negative.